src/main/java/io/phdata/streamliner/schemadefiner/util/StreamlinerUtil.java

```diff
@@ -452,4 +452,47 @@ public static void createFile(String... files) {
               new String[] {
                 "VARCHAR", "CHAR", "CHARACTER", "STRING", "TEXT", "BINARY", "VARBINARY"
               }));
+
+  public static Configuration createConfig(String directory, Configuration ingestConfig, String param) {
+    if (directory == null) {
+      throw new RuntimeException(String.format("%s path is null.", param));
+    }
+    File f = new File(directory);
+    if (f.exists()) {
+      if (f.isFile()) {
+        throw new RuntimeException(
+            String.format(
+                "Arg: %s. Expected directory. Found file: %s",
+                    param,
+                    directory));
+      }
+    } else {
+      throw new RuntimeException(String.format("%s does not exists. Path: %s", param, directory));
+    }
+      Configuration tempConf;
+    if (ingestConfig == null) {
+      tempConf = new Configuration();
+    } else {
+      tempConf =
+          new Configuration(
+              ingestConfig.getName(),
+              ingestConfig.getEnvironment(),
+              ingestConfig.getPipeline(),
+              ingestConfig.source,
+              ingestConfig.getDestination());
+    }
+
+    File tablesConfig[] = f.listFiles();
+    Arrays.asList(tablesConfig).stream()
+        .forEach(
+            tableConfig -> {
+                /* state-directory have diff.yml and table configs like EMPLOYEE.yml, DEPT.yml.
+                * To create a common config file, diff.yml should not be read.*/
+              if (!tableConfig.getName().equals("diff.yml")) {
```

Comment:
I think diff.yml is going to be used in multiple locations. Let's move that to a constant.

I also think that a table named "diff" is possible. Perhaps we call it "streamliner-diff" which is less likely.